docs/variants/msi_z790/recovery.md

```diff
@@ -0,0 +1,252 @@
+# Recovery
+
+## Intro
+
+This project is in early development phase. On certain
+hardware configurations, the Dasharo firmware may not boot correctly (i.e.
+we will have "bricked" the platform). In such a case, the recovery procedure can
+reinstall the original firmware from the board manufacturer.
+
+There are two documented recovery methods: using a [CH341A programming kit](https://3mdeb.com/shop/modules/ch341a-flash-bios-usb-programmer-kit-soic8-sop8/)
+or an [RTE](https://3mdeb.com/shop/open-source-hardware/rte/).
+
+### External flashing with programmer
+
+#### RTE
+
+In this case, using external programmer is necessary. We are using
+[RTE](https://3mdeb.com/open-source-hardware/#rte)
+here.
+
+* Connect programmer to the flash chip as shown in the
+  [Hardware connection / SPI](../development/#hardware-connection) section of
+  the `Development` documentation.
+
+* Download official BIOS from vendor's website (this is the newest version, you
+  may choose an older one too or in the best case use your firmware backup):
+
+```bash
+<<<<<<< HEAD
+wget https://download.msi.com/bos_exe/mb/7D25v13.zip
+unzip 7D25v13.zip
+=======
+wget https://download.msi.com/bos_exe/mb/7E06vA3.zip
+unzip 7E06vA3.zip
+>>>>>>> dc0451a6 (docs/variants/msi_z790: add initial documentation)
+```
+
+* Flash via external programmer:
+
+> The command line will be different, depending on the programmer you use
+
+```bash
+<<<<<<< HEAD
+flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v13/E7D25IMS.130
+=======
+flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7E06vA3/E7E06IMS.A30
+>>>>>>> dc0451a6 (docs/variants/msi_z790: add initial documentation)
```

Comment:
please fix

Reply:
Fixed by force pushing to 544d914b198e28f21f2d8ef088f2425d656c0836